### PR TITLE
The Great Uniform Overhaul

### PR DESCRIPTION
**Pull request by scarwolff**
## About The Pull Request

Redoes every single legacy uniform (specifically in uniform.dmi) sprite in the game to better fit out style and not be shit, also organises the dmis and code to be a lot nicer to navigate, also does a minor tweak so wasters have more thematic stuff.

image

credit to beestation for this format 

image

**I dream a future ... a future where all Legacy Sprites in the main files are eradicated by my goddamn hand.**

### CODE REVIEW

**Comment by Hekzder** [comment]
Okay.... this is epic. Very nice with the organization in the .dmi with the placeholders. Makes me a little bit hard in fact. Just a little bit.

**Review by Hekzder — CHANGES_REQUESTED** [review]
rll tawk though this doesn't compile. Mammoth.dmm needs to be edited to change the path of or otherwise remove the now renamed/removed clothing items that were previously pre-placed on the map.

**Review by Hekzder — APPROVED** [review]
Compiles and generally seems fine. Sprites are neat and this is a pretty big step in terms of replacing our legacy sprites with actually good sprites. Fine with me.

File: mojave/jobs/job_types/wasteland/wastelander.dm [hunk 1]
```diff
@@ -44,33 +44,49 @@
...
-		/obj/item/clothing/suit/ms13/trench,\
-		/obj/item/clothing/suit/ms13/trench/dapper,\
-		/obj/item/clothing/suit/ms13/ljacket/veteran,\
-        /obj/item/clothing/suit/armor/ms13/leatherarmor/light)
+		/obj/item/clothing/under/ms13/wasteland/caravaneer,\
+		/obj/item/clothing/under/ms13/wasteland/machinist,\
+		/obj/item/clothing/under/ms13/wasteland/lumberjack,\
+		/obj/item/clothing/under/ms13/wasteland/mercc,\
+		/obj/item/clothing/under/ms13/wasteland/roving,\
+		/obj/item/clothing/under/ms13/mechanicprewar,\
```
Comment (ProfessorPopoff) [inline mojave/jobs/job_types/wasteland/wastelander.dm:72]: image
This one is a bit busted.

Comment (scarwolff) [reply mojave/jobs/job_types/wasteland/wastelander.dm]: fixed

Comment (scarwolff) [reply mojave/jobs/job_types/wasteland/wastelander.dm]: good catch

Comment (ProfessorPopoff) [reply mojave/jobs/job_types/wasteland/wastelander.dm]: I didn't see any other suits other than this one that were screwed up. Should all be good.

**Comment by ProfessorPopoff** [comment]
Nice compile bud.

**Comment by scarwolff** [comment]
thas a byeah from me, ready to roll now foo

**Pull request merged**